data/showcase/aides_jeunes.yml

```diff
@@ -0,0 +1,14 @@
+title: Aides Jeunes
+description:
+  fr: Calculer l'éligibilité et les montants de nombreuses aides sociales.
+  en: ??
```

Comment:
Il semblerait que je n'ai pas les droits pour fait le commti moi même.
Pour la description anglaise, je propose : 
`Calculate entitlement and amounts for many social benefits.`

Qu'en penses tu @guillett ?